vm/src/pyobject.rs

```diff
@@ -1016,6 +1016,10 @@ pub trait PyValue: fmt::Debug + PyThreadingConstraint + Sized + 'static {
             )))
         }
     }
+
+    fn into_pyresult_ref_with_type(self, vm: &VirtualMachine, cls: PyTypeRef) -> PyResult {
```

Comment:
`ref` in `into_ref` means `PyRef`. So `into_pyresult_with_type` will be enough here. `into_pyobject_with_type` would be also good because `into_pyobject` will be always successful

Reply:
i get the first one, `into_pyresult_with_type`! Thank you!